### PR TITLE
Fix IE compat issue with default function params

### PR DESCRIPTION
**Pull request by shadyvb**
humanmade/altis-analytics#136

### CODE REVIEW

File: src/wp-consent-api.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ function consent_api_set_cookie(name, value, path = '/') {
 
     if (window.location.protocol !== "https:") secure = '';
 
-    document.cookie = name + "=" + value + secure + expires + ";path=" + path;
+    document.cookie = name + "=" + value + secure + expires + ";path=" + ( path !== undefined ? path : '/' );
```
Comment (roborourke) [inline src/wp-consent-api.js:66]: I think `( path || '/' )` would also work.

**Review by roborourke — APPROVED** [review]

**Comment by roborourke** [comment]
Check the version constraint upstream, hopefully it's tracking the patch version already but we should do a patch release of this at any rate.

File: src/wp-consent-api.js (hunk 1 above)
Comment (shadyvb) [reply src/wp-consent-api.js:66]: Yeah, I just opted to be explicitly checking if the value is provided or not, regardless of the fact that any falsey value wouldn't work.

**Comment by shadyvb** [comment]
@roborourke Do you mean https://github.com/humanmade/altis-consent/blob/main/composer.json ? I can prep a patch release if we want to, yes.

**Comment by roborourke** [comment]
Yeah if you make a new patch release here then the upstream `altis/consent` and `altis/privacy` packages will be able to get this update. No need to explicitly do anything more.

**Comment by shadyvb** [comment]
Ah, okay, cool. Will do. Can you refer me to how the script minifcation process was usually done here ? I had to manually update the file. Just so I can add it to the docs.

**Comment by roborourke** [comment]
I don't think this repo ever had any automation for script minification so it's a manual job at the moment 👍

**Pull request merged**